templates/IAM/snowflake-bucket-access.j2

```diff
@@ -15,33 +11,25 @@ Resources:
   SnowflakeServicePolicy:
     Type: 'AWS::IAM::ManagedPolicy'
     Properties:
-      PolicyDocument: !Sub |
-        {
-          "Version": "2012-10-17",
-          "Statement": [
-            {
-              "Effect": "Allow",
-              "Action": [
-                "s3:GetObject",
-                "s3:GetObjectVersion"
-              ],
-              "Resource": "${BucketArn}/${BucketPrefix}/*"
-            },
-            {
-              "Effect": "Allow",
-              "Action": [
-                "s3:ListBucket",
-                "s3:GetBucketLocation"
-              ],
-              "Resource": "${BucketArn}",
-              "Condition": {
-                "StringLike": {
-                  "s3:prefix": "${BucketPrefix}/*"
-                }
-              }
-            }
-          ]
-        }
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+{% for bucket in sceptre_user_data.Buckets %}
+          - Effect: Allow
+            Action:
+              - 's3:GetObject'
+              - 's3:GetObjectVersion'
+            Resource:
+              - "{{ bucket.Arn }}/{{ bucket.Prefix }}/*"
```

Comment:
Do we need a list here? Below we do not use it?

Reply:
good catch.  i have reverted it back to a string.